scripts/local_testnet/README.md

```diff
@@ -1,201 +1,70 @@
 # Simple Local Testnet
 
-These scripts allow for running a small local testnet with multiple beacon nodes and validator clients and a geth execution client.
+These scripts allow for running a small local testnet with a default of 4 beacon nodes, 4 validator clients and 4 geth execution clients using Kurtosis.
 This setup can be useful for testing and development.
 
-## Requirements
+## Installation
 
-The scripts require `lcli`, `lighthouse`, `geth`, `bootnode` to be installed on `PATH` (run `echo $PATH` to view all `PATH` directories).
+1. Install [Docker](https://docs.docker.com/get-docker/). Verify that Docker has been successfully installed by running `sudo docker run hello-world`. 
 
+1. Install [Kurtosis](https://docs.kurtosis.com/install/). Verify that Kurtosis has been successfully installed by running `kurtosis version` which should display the version.
 
-MacOS users need to install GNU `sed` and GNU `grep`, and add them both to `PATH` as well.
+1. Install [yq](https://github.com/mikefarah/yq). If you are on Ubuntu, you can install `yq` by running `sudo apt install yq -y`.
 
-The first step is to install Rust and dependencies. Refer to the [Lighthouse Book](https://lighthouse-book.sigmaprime.io/installation-source.html#dependencies) for installation. We will also need [jq](https://jqlang.github.io/jq/), which can be installed with `sudo apt install jq`.
+## Starting the testnet
 
-Then, we clone the Lighthouse repository:
-```bash
-cd ~
-git clone https://github.com/sigp/lighthouse.git
-cd lighthouse
-```
-We are now ready to build Lighthouse. Run the command:
+To start a testnet, from the Lighthouse root repository:
 
 ```bash
-make
-make install-lcli
+cd ./scripts/local_testnet
+./start_local_testnet.sh
 ```
 
-This will build `lighthouse` and `lcli`. For `geth` and `bootnode`, go to [geth website](https://geth.ethereum.org/downloads) and download the `Geth & Tools`. For example, to download and extract `Geth & Tools 1.13.1`:
-
-```bash
-cd ~
-curl -LO https://gethstore.blob.core.windows.net/builds/geth-alltools-linux-amd64-1.13.1-3f40e65c.tar.gz
-tar xvf geth-alltools-linux-amd64-1.13.1-3f40e65c.tar.gz
-```
+It will build Lighthouse and will take an approximately 12 minutes to complete. Once built, the testing will be started automatically. You will see a list of services running and "Started!" at the end. 
```

Comment:
```suggestion
It will build a Lighthouse docker image from the root of the directory and will take an approximately 12 minutes to complete. Once built, the testing will be started automatically. You will see a list of services running and "Started!" at the end. 
You can also select your own Lighthouse docker image to use by specifying it in `network_params.yml` under the `cl_image ` key.
Full configuration reference for kurtosis is specified here [here](https://github.com/kurtosis-tech/ethereum-package?tab=readme-ov-file#configuration)
```